client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -52,11 +59,23 @@ class PeopleListSectionHeader extends Component {
 		return null;
 	}
 
+	isSubscribersTab() {
+		const { currentRoute } = this.props;
+
+		return startsWith( currentRoute, '/people/email-followers' );
+	}
+
 	render() {
 		const { label, count, children, translate } = this.props;
 		const siteLink = this.getAddLink();
+		const addSubscriberLink = this.getAddSubscriberLink();
 		const classes = classNames( this.props.className, 'people-list-section-header' );
 
+		const showInviteUserBtn =
+			( siteLink && ! this.isSubscribersTab() ) || ! isEnabled( 'subscriber-importer' );
```

Comment:
This logic will allow the invite button to appear as long as the `subscriber-importer` feature flag is not enabled. Is this expected behavior?

Reply:
Yes, that's expected behavior. Instead of the Invite button, we are going to show `Add Subscriber` button when the flag is enabled.

Reply:
Got it. Just to make sure I understand correctly, the invite user button will be displayed if the feature flag is disabled even `siteLink` is null, right?